src/app/updater.js

```diff
@@ -1,7 +1,7 @@
 (function (App) {
     'use strict';
     var DHT = require('bittorrent-dht');
-    var ed = require('@noble/ed25519'); // better use ed25519-supercop but need rebuild ed25519 for windows
+    var ed = require('bittorrent-dht-sodium'); // better use ed25519-supercop but need rebuild ed25519 for windows
```

Comment:
> better use ed25519-supercop but need rebuild ed25519 for windows

remove this comment?

`bittorrent-dht-sodium` is based on [libsodium](https://github.com/jedisct1/libsodium)

> Sodium supports a variety of compilers and operating systems, including Windows (with MingW or Visual Studio, x86 and x64), iOS, Android, as well as Javascript and Webassembly.

Reply:
We just used that library because there was some problem with Windows, but if everything is fine now, then yes, the standard one is better